scipy/interpolate/tests/test_polyint.py

```diff
@@ -408,5 +417,106 @@ def test_all_zeros(self):
         assert_equal(pch(xx), 0.)
 
 
+class TestCubicSpline(object):
```

Comment:
As far as I understand, `TestCase` is not necessary with `nosetests`. I just figured that `TestCase` is from `unittest` standard module, and it is supposed to used in a different way.

So it seems more logical not to inherit from `TestCase`.